client/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 // import { Button, AppBar, Box, Toolbar, IconButton, Grid, Typography } from '@mui/material';
+import settings from '../public/settings.png';
+import intime from '../public/intime.png';
+import data from '../public/data.png';
+import barchart from '../public/barchart.png';
+import gear from '../public/gear.png';
+// import mockChart from '../public/mockChart.jpg';
 import '../public/styles.css';
 
 export default function HomePage() {
+    // this is a temp solution to hide the navbar. Final one will hide on scroll up
```

Comment:
Create an issue for this or add a `FIX ME` flag so that this doesn't get forgotten.

Reply:
Done!